FinSearchUnified/Bundle/SearchBundleFindologic/QueryBuilderFactory.php

```diff
@@ -250,4 +250,33 @@ public function createQueryBuilder()
 
         return $querybuilder;
     }
+
+    /**
+     * @param Criteria $criteria
+     * @param ShopContextInterface $context
+     *
+     * @return QueryBuilder
+     * @throws Exception
+     */
+    public function createSearchNavigationQuery(Criteria $criteria, ShopContextInterface $context)
```

Comment:
Explicitly mention that no additional filters are sent, maybe even in the method name itself.